.docker/autoline/backend.Dockerfile

```
@@ -0,0 +1,21 @@
+FROM node:16.16.0-slim as backend-build
+
+RUN apt-get update
+RUN apt-get install -y openssl
+
+WORKDIR /app
+
+COPY ./package*.json ./
```

We need to copy `yarn.lock`
```suggestion
COPY ./package.json ./
COPY ./yarn.lock ./
```

---

Got you